setup.py

```diff
@@ -52,7 +52,7 @@ def get_all_extras() -> Dict:
     "pandas<1.4,>=1.3.4",
     "watchdog >=2.1.6",
     "pytest==7.0.0",
-    "docker==5.0.3"
```

Comment:
Can we maybe instead have this approach for Windows only and otherwise use the pypi distribution?

Reply:
Good point, we can do it quite easily in `setup.py`; less easily in every testenv in `tox.ini` (but note that we install `docker` via `setup.py` in tox test environments; the ones we care for CI)

Reply:
[9ea2226](https://github.com/valory-xyz/open-autonomy/pull/1144/commits/9ea222662fb6741cabdd74fac5d2794e3b0b0ba1)